### PR TITLE
change redirecting user after revert to the original file

### PR DESCRIPTION
**Pull request by tomfelix-apptension**

### CODE REVIEW

File: frontend/schemaCMS/src/modules/dataSource/dataSource.sagas.js [hunk 1]
```diff
@@ -297,7 +298,11 @@ function* revertToJob({ payload: { dataSourceId, jobId } }) {
     const { data } = yield api.post(`${DATA_SOURCES_PATH}/${dataSourceId}/revert-job`, { id: jobId });
 
     yield put(DataSourceRoutines.revertToJob.success(data));
-    browserHistory.push(`/datasource/${dataSourceId}/${RESULT_PAGE}`);
+    if (data.activeJob.scripts.length) {
+      browserHistory.push(`/datasource/${dataSourceId}/${RESULT_PAGE}`);
+    } else {
+      browserHistory.push(`/datasource/${dataSourceId}/${PREVIEW_PAGE}`);
+    }
```
Comment (cpBurn) [inline frontend/schemaCMS/src/modules/dataSource/dataSource.sagas.js:305]: We need to add test for this

Comment (tomfelix-apptension) [reply frontend/schemaCMS/src/modules/dataSource/dataSource.sagas.js:305]: added

**Review by cpBurn — APPROVED** [review]

**Comment by cpBurn** [comment]
@tomfelix-apptension can you add Changelog please?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.7.0 (August 27, 2020) - Fixes
```
Comment (cpBurn) [inline CHANGELOG.md:1]: Let's do like that:

```
## [Unreleased]

### Fixes
```

Comment (cpBurn) [reply CHANGELOG.md:1]: sorry for not mentioning, but we decided that's it's better to have release branches, so we just add what you fixed under ### Fixes and we add the version later

Comment (tomfelix-apptension) [reply CHANGELOG.md:1]: ok

**Pull request merged**